packages/neuron-wallet/src/services/addresses.ts

```diff
@@ -201,7 +201,7 @@ export default class AddressService {
     await getConnection().manager.save(publicKeyInfo)
 
     const addressMeta = AddressMeta.fromHdPublicKeyInfoModel(publicKeyInfo.toModel())
-    this.notifyAddressCreated([addressMeta], undefined)
+    AddressService.notifyAddressCreated([addressMeta], undefined)
```

Comment:
`generateAndSaveForPublicKey` is a static function, and a static function calls it. If use `this`, when I start Neuron the first time without a database file, it throws an error that can not called with undefined.